src/simsopt/geo/surfaceobjectives.py

```diff
@@ -16,7 +16,18 @@ class Area(Optimizable):
     Wrapper class for surface area label.
     """
 
-    def __init__(self, surface):
+    def __init__(self, in_surface, range=None, nphi=None, ntheta=None):
+        if range is None:
+            if in_surface.stellsym:
+                range = Surface.RANGE_HALF_PERIOD
+            else:
+                range = Surface.RANGE_FIELD_PERIOD
+        if nphi is None:
+            nphi = len(in_surface.quadpoints_phi)
+        if ntheta is None:
+            ntheta = len(in_surface.quadpoints_theta)
+        surface = in_surface.__class__.from_nphi_ntheta(nphi=nphi, ntheta=ntheta, range=range, nfp=in_surface.nfp, stellsym=in_surface.stellsym, \
```

Comment:
Do we really need to force the creation of a new surface object here? It seems that this is only necessary when 
`range is not None or nphi is not None or ntheta is not None`. Same for the objectives below.

Reply:
@florianwechsung done
@landreman this was a simple change to test out the new shared dofs functionality with the geometry objects